docker/web/dev/entrypoint.sh

```diff
@@ -14,6 +14,6 @@ su www-data bash -c "cd /var/www/script/migration && composer install --no-dev &
 su www-data bash -c "composer install -d /var/www"
 
 su www-data bash -c "cd /var/www && yarn install"
-su www-data bash -c "cd /var/www && yarn encore dev"
+su www-data bash -c "cd /var/www && yarn encore dev --watch &"
```

Comment:
May be we should have both
```
su www-data bash -c "cd /var/www && yarn encore dev
su www-data bash -c "cd /var/www && yarn encore dev --watch &
```

If we have only `su www-data bash -c "cd /var/www && yarn encore dev --watch &` it may fail and not be noticed.
If `su www-data bash -c "cd /var/www && yarn encore dev` fail, then build will fail.

Reply:
Agree